### ISSUE TITLE
SaveAs() not working

### ISSUE THREAD

**Issue by mikjov93** [ShapeCrawler/ShapeCrawler#237]
SaveAs() not working. Throw error closed stream. Also template is changing instead just download changes in new file

**Comment by adamshakhabov** [comment]
@mikjov93, regarding "Cannot access a closed Stream", could you provide a sample that can be used "As Is". I couldn't reproduce this issue. For example, the code below is worked:

```C#
using var presentation = SCPresentation.Open(@"c:\temp\cantAccessRepro.pptx", true);
var title = presentation.Slides[0].Shapes.GetByName<IAutoShape>("Title 1");
title.TextBox!.Text = "New Title";

var ms = new MemoryStream();
presentation.SaveAs(ms);
```

Also, please provide your Target Framework version and check that you use the latest ShapeCrawler version.

**Comment by adamshakhabov** [comment]
`SaveAs()` bug was fixed in [**v0.30.0**](https://github.com/ShapeCrawler/ShapeCrawler/releases/tag/v0.30.0) release.

**Comment by sergio-poliedro** [comment]
I can't save the presentation since I open it as a template, and when the presentation is filled with info, is downloaded by the user and the original presentation remains intact. 

I can't figure how to create a duplicate either in order to delete it after the opreation, that might be a solution as well.
